CMakeLists.txt

```diff
@@ -98,6 +112,22 @@ endif()
 
 find_package(PkgConfig REQUIRED)
 
+############# Set AWS CRT based variables #############
+if(BUILD_AWS_CRT)
+    if(APPLE)
+        set(CFLIBRARY "-framework CoreFoundation")
+    endif()
+    set(AWS_C_COMMON_LIBRARY "aws-c-common")
+    set(AWS_LC_LIBRARY "AWSLC")
+    set(AWS_S2N_TLS_LIBRARY "s2n")
+    set(AWS_C_CAL_LIBRARY "aws-c-cal")
+    set(AWS_C_IO_LIBRARY "aws-c-io")
+    set(AWS_C_COMPRESSION_LIBRARY "aws-c-compression")
+    set(AWS_C_HTTP_LIBRARY "aws-c-http")
+    set(AWS_SDK_UTILS_LIBRARY "aws-c-sdkutils")
```

Comment:
To be consistent with the others can we name this variable `AWS_C_SDK_UTILS_LIBRARY`

Reply:
Done.